keyboards/enviousdesign/delirium/rev1/info.json

```diff
@@ -0,0 +1,305 @@
+                {"label": "&", "matrix": [1, 7], "x": 7, "y": 1.5},
+                {"label": "*", "matrix": [1, 8], "x": 8, "y": 1.5},
+                {"label": "(", "matrix": [1, 9], "x": 9, "y": 1.5},
+                {"label": ")", "matrix": [1, 10], "x": 10, "y": 1.5},
+                {"label": "_", "matrix": [1, 11], "x": 11, "y": 1.5},
+                {"label": "+", "matrix": [1, 12], "x": 12, "y": 1.5},
+                {"label": "Backspace", "matrix": [1, 14], "x": 13, "y": 1.5, "w": 2},
+                {"label": "Insert", "matrix": [1, 15], "x": 15.25, "y": 1.5},
+                {"label": "Home", "matrix": [1, 16], "x": 16.25, "y": 1.5},
+                {"label": "PgUp", "matrix": [1, 17], "x": 17.25, "y": 1.5},
+                {"label": "Tab", "matrix": [2, 0], "x": 0, "y": 2.5, "w": 1.5},
+                {"label": "Q", "matrix": [2, 2], "x": 1.5, "y": 2.5},
+                {"label": "W", "matrix": [2, 3], "x": 2.5, "y": 2.5},
+                {"label": "E", "matrix": [2, 4], "x": 3.5, "y": 2.5},
+                {"label": "R", "matrix": [2, 5], "x": 4.5, "y": 2.5},
+                {"label": "T", "matrix": [2, 6], "x": 5.5, "y": 2.5},
+                {"label": "Y", "matrix": [2, 7], "x": 6.5, "y": 2.5},
+                {"label": "U", "matrix": [2, 8], "x": 7.5, "y": 2.5},
+                {"label": "I", "matrix": [2, 9], "x": 8.5, "y": 2.5},
+                {"label": "O", "matrix": [2, 10], "x": 9.5, "y": 2.5},
+                {"label": "P", "matrix": [2, 11], "x": 10.5, "y": 2.5},
+                {"label": "{", "matrix": [2, 12], "x": 11.5, "y": 2.5},
+                {"label": "}", "matrix": [2, 13], "x": 12.5, "y": 2.5},
+                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 2.5, "w": 1.25, "h": 2},
+                {"label": "Delete", "matrix": [2, 15], "x": 15.25, "y": 2.5},
+                {"label": "End", "matrix": [2, 16], "x": 16.25, "y": 2.5},
+                {"label": "PgDn", "matrix": [2, 17], "x": 17.25, "y": 2.5},
+                {"label": "Caps Lock", "matrix": [3, 0], "x": 0, "y": 3.5, "w": 1.75},
+                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 3.5},
+                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 3.5},
+                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 3.5},
+                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 3.5},
+                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 3.5},
+                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 3.5},
+                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 3.5},
+                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 3.5},
+                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 3.5},
+                {"label": ":", "matrix": [3, 11], "x": 10.75, "y": 3.5},
+                {"label": "@", "matrix": [3, 12], "x": 11.75, "y": 3.5},
+                {"label": "~", "matrix": [3, 13], "x": 12.75, "y": 3.5},
```

Comment:
Moves Enter key to the Home Row.

(Required for `tkl_iso` Community Layout compatibility.)

```suggestion
                {"label": "Delete", "matrix": [2, 15], "x": 15.25, "y": 2.5},
                {"label": "End", "matrix": [2, 16], "x": 16.25, "y": 2.5},
                {"label": "PgDn", "matrix": [2, 17], "x": 17.25, "y": 2.5},
                {"label": "Caps Lock", "matrix": [3, 0], "x": 0, "y": 3.5, "w": 1.75},
                {"label": "A", "matrix": [3, 2], "x": 1.75, "y": 3.5},
                {"label": "S", "matrix": [3, 3], "x": 2.75, "y": 3.5},
                {"label": "D", "matrix": [3, 4], "x": 3.75, "y": 3.5},
                {"label": "F", "matrix": [3, 5], "x": 4.75, "y": 3.5},
                {"label": "G", "matrix": [3, 6], "x": 5.75, "y": 3.5},
                {"label": "H", "matrix": [3, 7], "x": 6.75, "y": 3.5},
                {"label": "J", "matrix": [3, 8], "x": 7.75, "y": 3.5},
                {"label": "K", "matrix": [3, 9], "x": 8.75, "y": 3.5},
                {"label": "L", "matrix": [3, 10], "x": 9.75, "y": 3.5},
                {"label": ":", "matrix": [3, 11], "x": 10.75, "y": 3.5},
                {"label": "@", "matrix": [3, 12], "x": 11.75, "y": 3.5},
                {"label": "~", "matrix": [3, 13], "x": 12.75, "y": 3.5},
                {"label": "Enter", "matrix": [3, 14], "x": 13.75, "y": 2.5, "w": 1.25, "h": 2},
```

Reply:
This request is still pending.